### PR TITLE
Fix/reduce station tolerance

### PR DESCRIPTION
**Pull request by floweisshardt**
fixes parts of https://github.com/ipa320/msh/issues/697

### CODE REVIEW

File: cob_default_robot_behavior/scripts/trigger_srvs_cob4-7.py [hunk 1]
```diff
@@ -89,6 +89,57 @@ def front_to_home_cb(req):
         sss.move("torso","home",False)
     else:
         return TriggerResponse(False, "Could not move arms.")
+
+    return TriggerResponse(True, "")
+
+def pick_cb(req):
```
Comment (NadiaHG) [inline cob_default_robot_behavior/scripts/trigger_srvs_cob4-7.py:95]: please delete this, cob4-7 hasn't got arms

File: cob_hardware_config/cob4-7/config/base_driver.yaml [hunk 2]
```diff
@@ -28,7 +28,7 @@ nodes:
     id: 1
     dcf_overlay: # "ObjectID" : "ParameterValue" (both as strings)
       "6098": "19" # homing method
-      "607C": "-48531" #"-52642" #"-54359" home offset
+      "607C": "-48238.5" #"-52642" #"-54359" home offset
```
Comment (NadiaHG) [inline cob_hardware_config/cob4-7/config/base_driver.yaml:31]: did you re-calibrate the base?

**Review by NadiaHG — CHANGES_REQUESTED** [review]

**Comment by floweisshardt** [comment]
sorry, seems to be a bad base. I'll close and reopen against ipa320/indigo_dev

**Comment by floweisshardt** [comment]
sorry, seems to be a bad base. I'll close and reopen against ipa320/indigo_dev

**Pull request closed without merging**